src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public aSide: number,
+    public bSide: number,
+    public cSide: number,
+  ) {
+    const sides = [aSide, bSide, cSide].sort((a: number, b: number) => a - b);
+
+    if ((sides[0] + sides[1]) <= sides[2] || sides.find((el) => el <= 0)) {
+      throw new Error('wrong values for triangle');
```

Comment:
It would be great to specify why exactly they're wrong
![image](https://github.com/mate-academy/ts_figures_classes/assets/96409563/fec9685b-2397-4ebb-8f1d-55c5cec6963f)